js/js-variables-and-numbers_simple-calculator/js/index.js

```diff
@@ -61,24 +73,36 @@ const divideByTwoButton = document.querySelector('[data-js="divide-by-two"]');
 
 increaseByOneButton.addEventListener("click", () => {
   // 7. Increase the value of "operand1" by 1. Log the new value to the console.
+  const increaseByOne = operand1 + 1;
+  console.log("Increase first operand (12) by 1:", increaseByOne);
 });
 
 increaseByFiveButton.addEventListener("click", () => {
   // 8. Increase the value of "operand1" by 5. Log the new value to the console.
+  const increaseByFive = operand1 + 5;
```

Comment:
Here I think you would need to use `+= 5` to make sure 5 is added every time you press the add 5 button, otherwise it will show always the same number + 5, I think?